cds_ils/cli.py

```diff
@@ -294,69 +292,94 @@ def create_demo_items(items_path):
 @with_appcontext
 def create_loan(user_email, is_past_loan):
     """Create a loan."""
-    patron = User.query.filter_by(email=user_email).one()
-    patron_pid = patron.get_id()
+    ongoing_loan_item_pid = "vgrh9-jvj8E"
+    past_loan_item_pid = "678e3-an678A"
 
-    loc_pid, _ = current_app_ils.get_default_location_pid
+    active_loan = (
+        get_active_loan_by_item_pid(
+            {"type": "pitmid", "value": ongoing_loan_item_pid}
+        )
+        .execute()
+        .hits
+    )
 
-    delivery = list(
-        current_app.config["ILS_CIRCULATION_DELIVERY_METHODS"].keys()
-    )[randint(0, 1)]
+    lt_es7 = ES_VERSION[0] < 7
+    total = active_loan.total if lt_es7 else active_loan.total.value
 
-    loan_dict = {
-        "pid": RecordIdProviderV2.create().pid.pid_value,
-        "patron_pid": "{}".format(patron_pid),
-        "pickup_location_pid": "{}".format(loc_pid),
-        "transaction_location_pid": "{}".format(loc_pid),
-        "transaction_user_pid": "{}".format(patron_pid),
-        "delivery": {"method": delivery},
-    }
-
-    if is_past_loan:
-        loan_dict["state"] = "ITEM_RETURNED"
-        loan_dict["document_pid"] = "qaywb-gfe4B"
-        transaction_date = start_date = arrow.utcnow() - timedelta(days=365)
-        end_date = start_date + timedelta(weeks=4)
-        item_pid = "678e3-an678A"
+    if total > 0 and not is_past_loan:
+        click.secho(
+            "Item for ongoing loan is already loaned by patron with email {0}."
+            .format(
+                active_loan[0].patron.email
+            ),
+            fg="red",
+        )
     else:
-        loan_dict["state"] = "ITEM_ON_LOAN"
-        loan_dict["document_pid"] = "67186-5rs9E"
-        transaction_date = start_date = arrow.utcnow() - (
-            timedelta(weeks=4)
-            - timedelta(
-                days=current_app.config[
-                    "ILS_CIRCULATION_LOAN_WILL_EXPIRE_DAYS"
-                ]
+        patron = User.query.filter_by(email=user_email).one()
+        patron_pid = patron.get_id()
+
+        loc_pid, _ = current_app_ils.get_default_location_pid
+
+        delivery = list(
+            current_app.config["ILS_CIRCULATION_DELIVERY_METHODS"].keys()
+        )[randint(0, 1)]
+
+        loan_dict = {
+            "pid": RecordIdProviderV2.create().pid.pid_value,
+            "patron_pid": "{}".format(patron_pid),
+            "pickup_location_pid": "{}".format(loc_pid),
+            "transaction_location_pid": "{}".format(loc_pid),
+            "transaction_user_pid": "{}".format(patron_pid),
+            "delivery": {"method": delivery},
+        }
+
+        if is_past_loan:
+            loan_dict["state"] = "ITEM_RETURNED"
+            loan_dict["document_pid"] = "qaywb-gfe4B"
```

Comment:
minor: we could take this PID on top for readability like you did with item pids